### DISCUSSION TITLE
Why would some PRs won't show up in GitHub API for search

### DISCUSSION THREAD

**Discussion by letsintegreat** [community/community#41706] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I want to list all the PRs that are merged, and have the label: `codepeak 22`.

Therefore, I am using this endpoint: https://api.github.com/search/issues?q=is%3Apr+is%3Amerged+label%3A%22codepeak+22%22

However, some PRs are not showing up, even after being merged and being labelled `codepeak 22`. I have noted that the label name is not case sensitive.

Example of PRs that are not being showed up -

- https://github.com/infinity1729/EventListing/pull/11
- https://github.com/mohit200008/FoodSaver20008/pull/55

What am I missing here?

**Comment by LangLangBart** [comment]
The link will return 30 results by default. The totalCount of your search results is ~`178`.
```zsh
gh api -X GET search/issues?q=is%3Apr+is%3Amerged+label%3A%22codepeak+22%22 --jq '.items | length'
# 30
```

The [GitHub search](https://docs.github.com/en/rest/search?apiVersion=2022-11-28#search-code) allows for some `Query parameters`, for example `per_page` or `page`.
```zsh
gh api -F per_page=100 -F page=1 -X GET search/issues?q=is%3Apr+is%3Amerged+label%3A%22codepeak+22%22 --jq '.items | length'
# 100
gh api -F per_page=100 -F page=2 -X GET search/issues?q=is%3Apr+is%3Amerged+label%3A%22codepeak+22%22 --jq '.items | length'
# 78
```

With [gh](https://github.com/cli/cli#installation) you can use the `--paginate` option which will give you all the results.
```zsh
gh api --paginate -X GET search/issues?q=is%3Apr+is%3Amerged+label%3A%22codepeak+22%22  --jq '.items | length'
# 100
# 78
```

Filtering the returned results for the two PR's from your description with [jq](https://stedolan.github.io/jq/manual/).
```zsh
gh api --paginate  -X GET search/issues?q=is%3Apr+is%3Amerged+label%3A%22codepeak+22%22 --jq '.items[] | select(.title == "Updated events UI design" or .title == "Fixed ingredients overflow issue") | "\(.number) \(.title) \t ⏰ \(.pull_request.merged_at)"'
# 11 Updated events UI design     ⏰ 2022-12-13T13:32:49Z
# 55 Fixed ingredients overflow issue     ⏰ 2022-12-09T10:36:45Z
```
